mypy/server/update.py

```diff
@@ -270,6 +270,20 @@ def update(self,
         self.previous_messages = messages[:]
         return messages
 
+    def trigger(self, target: str) -> List[str]:
+        """Trigger a specific target explicitly.
+
+        This is intended for use by the suggestions engine.
+        """
+        self.manager.errors.reset()
```

Comment:
What happens if there are already errors? (Perhaps unrelated to the suggestion target, or perhaps involved somehow?)

Reply:
They'll get regenerated, since the targets that caused them are in `self.previous_targets_with_errors`.

(This is the same as how it happens when doing a normal run.)